src/test/resources/feature/SignUp.feature

```diff
@@ -0,0 +1,54 @@
+@Irina
+Feature: SignUp
+
+  Background:
+    Given user is on "CREATE AN ACCOUNT" page
+
+  Scenario: SignUp with valid data - successful
```

Comment:
Would be possible to SignUp with **valid** data **unsuccessfully**?

Reply:
Done